apps/site/assets/ts/helpers/icon.tsx

```diff
@@ -107,3 +107,44 @@ export const caret = (
     />
   );
 };
+
+/* 
+A minimal component that gives its child component 
+a basic tooltip containing the specified text. Has no
+visual appearance itself.
+
+Note: data-selector="true" and data-original-title are
+specified here in order to preserve `title` text post-trigger
```

Comment:
If we can figure out which Bootstrap version we could remove this in, once we upgrade, it would be good to put that information here. Good comment as it stands though 👍

Reply:
This doesn't appear to the fixed in the latest version I tried, sadly. Will have to consider revisiting in the future...

Reply:
That's too bad 😢 Bootstrap, why do you make it hard to be accessible